src/js/msp/MSPHelper.js

```
@@ -2442,148 +2442,89 @@ MspHelper.prototype.setRawRx = function (channels) {
+                if (dataCompressionType === 0) {
+                    // Raw, uncompressed
+                    payloadView = new DataView(response.data.buffer, response.data.byteOffset + headerSize, dataSize);
+                    bytesCompressed = dataSize;
+                } else if (dataCompressionType === 1) {
+                    // Compressed
+                    const compressedCharCount = response.data.readU16();
+                    const compressedArray = new Uint8Array(
+                        response.data.buffer,
+                        response.data.byteOffset + headerSize + 2,
+                        dataSize - 2
+                    );
```

ESLint

[error] Missing trailing comma.

(comma-dangle)